rasa_nlu/test.py

```diff
@@ -803,19 +806,26 @@ def combine_entity_result(results, interpreter, data):
     return results
 
 
-def run_cv_evaluation(data: TrainingData,
-                      n_folds: int,
-                      nlu_config: RasaNLUModelConfig) -> CVEvaluationResult:
-    """Stratified cross validation on data
-    :param data: Training Data
-    :param n_folds: integer, number of cv folds
-    :param nlu_config: nlu config file
-    :return: dictionary with key, list structure, where each entry in list
+def cross_validate(data: TrainingData, n_folds: int,
+                   nlu_config: Union[RasaNLUModelConfig, Text]
+                   ) -> CVEvaluationResult:
+    """Stratified cross validation on data.
+
+    Args:
+        data: Training Data
+        n_folds: integer, number of cv folds
+        nlu_config: nlu config file
+
+    Returns:
+        dictionary with key, list structure, where each entry in list
               corresponds to the relevant result for one fold
     """
     from collections import defaultdict
     import tempfile
 
+    if isinstance(nlu_config, str):
```

Comment:
why is this sometimes a string?

Reply:
because users might want to pass in the filepath when calling `test(...)` instead of handling the config stuff before calling `test(...)`

Reply:
:+1: